spec/features/likes_features_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'allows users to like posts' do
+
+  before do
+    Post.create(caption: 'Here is my post')
+  end
+
+  scenario 'likes are initially 0 when post is created' do
+    sign_up
+    visit '/posts'
+    expect(page).to have_content('0 likes')
+  end
+
+  scenario 'a user can like a post, which increments the like count', js: true do
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)